_posts/en/pages/list/2016-03-10-announcements-join.md

```diff
@@ -9,7 +9,7 @@ version: 2
 ---
 Receive notification of important security announcements and releases for Bitcoin Core.
 
-Subscribe to the RSS feed below.
+Subscribe to the RSS feed below or add your email address to the [bitcoin-core-dev][] mailing list.
```

Comment:
Need to add the following to the bottom of the file to use the global reference links:

    {% include references.md %}